app/controllers/podcasts_controller.rb

```diff
@@ -14,7 +14,31 @@ def index
   def show
     @search_path = podcast_path(@podcast)
     @search_placeholder = 'Search episodes'
-    @episodes = @podcast.search_episodes params[:query]
+
+    case params[:sort]
+    when "old-first"
+      @sort = "old-first"
+    else
+      @sort = "new-first"
+    end
```

Comment:
This case can be replaced with a ternary operator @sort = params[:sort] == 'old-first' : 'new-first'